react/src/base/inputs/SprkCheckbox/SprkCheckbox.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+
+const SprkCheckbox = (props) => {
+  const {
+    children,
+    variant,
+    idString,
+    additionalClasses,
+    analyticsString,
+    additionalCheckboxClasses,
```

Comment:
Don't forget to update this to checkboxAdditionalClasses and same for the label one with our new standardization on naming.

Reply:
👍🏽